serverless/lambda/src/main/java/uk/gov/di/lambdas/SendNotificationHandler.java

```diff
@@ -114,7 +99,53 @@ public APIGatewayProxyResponseEvent handleRequest(
         }
     }
 
+    private APIGatewayProxyResponseEvent handleVerifyPhoneNumberRequest(
```

Comment:
might be possible to merge the following 2 methods

Reply:
@sanisc Fair point. I've updated!